UnityProject/Assets/Scripts/Items/Food/Edible.cs

```diff
@@ -176,9 +184,16 @@ public virtual void Eat(PlayerScript eater, PlayerScript feeder)
 			}
 			else
 			{
-				_ = Inventory.ServerDespawn(gameObject);
+				Bite(feeder, feederSlot);
```

Comment:
don't this still just duplicate the reagent mix?

Reply:
Are you talking about line 169? Please be more specific about what you're talking about

Reply:
"Stomach.StomachContents.Add(incomingFood.Clone());" 
yeah, this is called each time for each eat?

Reply:
So the content gets depplycaded?

Reply:
yes